ramls/permissions.raml

```diff
@@ -415,4 +415,17 @@ traits:
             body:
               text/plain:
                 example: "Internal server error"
-
+  /purge-deprecated:
+    post:
+      description: purge deprecated permissions
+      responses:
+        200:
+          description: "A list of permission names purged"
+          body:
+            application/json:
+              type: permissionNameListObject
+        500:
+          description: "Internal server error"
+          body:
+            text/plain:
+              example: "Internal server error"
```

Comment:
I would add 400 as a possible status here, so that you at least declare this as a possible outcome. Never know what RMB could do before your code is executed.

Reply:
Make sense. Added.